packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.kt

```diff
@@ -82,6 +82,16 @@ private const val MIN_EVENT_DT_MS: Long = 100
 private const val HEADER_KEY = "headers"
 private const val DIR_PERMISSIONS_REQUEST_CODE = 5394
 
+private fun slashifyFilePath(path: String?): String? {
```

Comment:
I would call it `normalizeFilePath`, but I'm also ok with slashify

Reply:
For this and the last, I was basically just stealing the function from [this](https://github.com/expo/expo/pull/12439/files#diff-764e029567150bffdda3518243fc2b67b050121e9effef1bfe8579e532637ffe) 😅